stdlib/public/Concurrency/CheckedContinuation.swift

```diff
@@ -300,11 +300,10 @@ public func withCheckedContinuation<T>(
 }
 
 @available(SwiftStdlib 5.1, *)
-@usableFromInline
 @_unsafeInheritExecutor // ABI compatibility with Swift 5.1
 @_unavailableInEmbedded
 @_silgen_name("$ss23withCheckedContinuation8function_xSS_yScCyxs5NeverOGXEtYalF")
-internal func __abi_withCheckedContinuation<T>(
+public func _unsafeInheritExecutor_withCheckedContinuation<T>(
```

Comment:
We've been pretty consistent with keeping all those methods "kept around for abi reasons" with the `__abi_` prefix, is there a real reason to change away from that here?

Reply:
It could be some __abi_unsafe... if you prefer, but it's easy to skim and notice why the hell such methods exist by looking at prefix, so I'd keep them tbh

Reply:
I can add more comments here. The `_unsafeInheritExecutor_` prefix is load-bearing in the awful, awful hack we're using to transitionally from `@_unsafeInheritExecutor`.

Reply:
The more comments help enough here, Lgtm thanks :)